### ISSUE TITLE
Tests fail on S3() instantiation 

### ISSUE THREAD

**Issue by Inveracity** [lemonsaurus/blackbox#36]
Pytest results:

```python
tests/test_storage.py::test_s3_handler_can_be_instantiated FAILED                                                [100%]

====================================================== FAILURES =======================================================
_________________________________________ test_s3_handler_can_be_instantiated _________________________________________

config_file = None

    def test_s3_handler_can_be_instantiated(config_file):
        """Test if the GoogleDrive storage handler can be instantiated."""
        from blackbox.handlers.storage import S3
>       S3()

tests\test_storage.py:11:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
blackbox\handlers\storage\s3.py:48: in __init__
    self.client = boto3.client(
.venv\lib\site-packages\boto3\__init__.py:93: in client
    return _get_default_session().client(*args, **kwargs)
.venv\lib\site-packages\boto3\session.py:258: in client
    return self._session.create_client(
.venv\lib\site-packages\botocore\session.py:827: in create_client
    endpoint_resolver = self._get_internal_component('endpoint_resolver')
.venv\lib\site-packages\botocore\session.py:700: in _get_internal_component
    return self._internal_components.get_component(name)
.venv\lib\site-packages\botocore\session.py:924: in get_component
    self._components[name] = factory()
.venv\lib\site-packages\botocore\session.py:163: in create_default_resolver
    endpoints = loader.load_data('endpoints')
.venv\lib\site-packages\botocore\loaders.py:132: in _wrapper
    data = func(self, *args, **kwargs)
.venv\lib\site-packages\botocore\loaders.py:420: in load_data
    found = self.file_loader.load_file(possible_path)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <botocore.loaders.JSONFileLoader object at 0x00000236ADF396D0>
file_path = .venv\\lib\\site-packages\\botocore\\data\\endpoints'

    def load_file(self, file_path):
        """Attempt to load the file path.

        :type file_path: str
        :param file_path: The full path to the file to load without
            the '.json' extension.

        :return: The loaded data if it exists, otherwise None.

        """
        full_path = file_path + '.json'
        if not os.path.isfile(full_path):
            return

        # By default the file will be opened with locale encoding on Python 3.
        # We specify "utf8" here to ensure the correct behavior.
        with open(full_path, 'rb') as fp:
>           payload = fp.read().decode('utf-8')
E           AttributeError: 'str' object has no attribute 'decode'

.venv\lib\site-packages\botocore\loaders.py:173: AttributeError
```

**Comment by lemonsaurus** [comment]
So, this probably happens because the `~/.aws` files don't exist. So maybe we can just somehow mock these files so that when `boto` tries to open it, it will get something that works?

We should probably use `moto` for this, too.

Here's what it expects to find:

At `~/.aws/config`, a file that looks like this:
```toml
[default]
region = eu-west-1
output = json
```

At `~/.aws/credentials`, a file that looks like this:
```toml
[default]
aws_access_key_id=ADJOCAMDAHCA
aws_secret_access_key=hc0#2mc5"dDA
```

**Comment by lemonsaurus** [comment]
Fixed by #54
